graph-db/serverless/src/utils/getConceptType.js

```diff
@@ -0,0 +1,17 @@
+const conceptTypes = {
+  C: 'collection',
+  G: 'granule',
+  S: 'service',
+  V: 'variable'
```

Comment:
Are there actually plans to index things like services and variables in the far future? I know we've seen things like ingest of new service revisions take painfully long when the service is associated with many collections - I wonder if graphdb tech can be applied there.

Reply:
In the near-term no, but it's certainly not out of the question for the future